programming/editors/pycharm.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: page
+title: PyCharm
+---
+
+# PyCharm Setup
+
+<div class="info">
+These instructions should also apply to IntelliJ IDEA with the Python plugin installed although the location of Python SDK settings will be different.
+</div>
+
+In order to take full advantage of PyCharm's smart assistance features, such as code completion, PyCharm needs to have access to the libraries used in your code.
+The following instructions will guide you through setting up your local environment to have access our `sr.robot3` API.
+
+1. Open your robot code project in PyCharm.
+2. Head to `File` &rarr; `Settings` or `PyCharm` &rarr; `Preferences` on a Mac.
```

Comment:
I suggest that we not use `code` style highlighting for menu items. For the VSCode article, the pattern was to **embolden** them instead. I realise this is a small thing, but we should probably be consistent.

The arrow is probably nicer than the ">" used in the VSCode article though, perhaps we should update that to use `&rarr;` too?

Reply:
I wonder if the arrow is more screen reader friendly

Reply:
41af49ad